src/components/atoms/Plugin/IFrame/hooks.ts

```diff
@@ -198,52 +135,36 @@ export default function useHook({
     setIFrameSize(w && h ? [w, h] : undefined);
   }, [width, height]);
 
+  const autoResizeScript = useMemo(() => {
+    return `<script id="_reearth_resize">
+      let inited = false;
+      if ("ResizeObserver" in window) {         
+        new window.ResizeObserver(entries => {
+          const win = document.defaultView;
+          const html = document.body.parentElement;
+          const st = win.getComputedStyle(html, "");
+          horizontalMargin = parseInt(st.getPropertyValue("margin-left"), 10) + parseInt(st.getPropertyValue("margin-right"), 10);
+          verticalMargin = parseInt(st.getPropertyValue("margin-top"), 10) + parseInt(st.getPropertyValue("margin-bottom"), 10);
+          const width = html.offsetWidth + horizontalMargin;
```

Comment:
Here i removed the conditions because the auto resize script will only inject once & should always get size from html.